### PR TITLE
feat(ShopCheckout): Add taxInPercent prop

### PR DESCRIPTION
**Pull request by lialila**
Story: https://trello.com/c/hekPRzbh/473-include-vat-in-shop-checkouts-on-website-3-8

### CODE REVIEW

File: src/slices/ShopCheckout/ShopCheckout.stories.tsx [hunk 1]
```diff
@@ -75,3 +75,16 @@ WithCHF.args = {
...
+
+export const WithTaxMessage = Template.bind({});
+WithTaxMessage.args = {
+  slice: {
+    title: 'Title',
+    batchId: 'batchId',
+    pricePerKg: 0.06,
+    initialContributionValue: 60,
+    checkoutText: 'Checkout Text',
+    taxInPercent: 8.4,
```
Comment (rschiffer) [inline src/slices/ShopCheckout/ShopCheckout.stories.tsx:87]: It doesn't really matter, but why `8.4`? The tax in CH is `8.1`, in AT it is `20`.

File: src/slices/ShopCheckout/ShopCheckout.test.tsx [hunk 2]
```diff
@@ -14,6 +14,7 @@ const defaultProps: ShopCheckoutProps = {
     batchId: 'batchId',
     pricePerKg: 0.06,
     initialContributionValue: 60,
+    taxInPercent: 8.4,
```
Comment (rschiffer) [inline src/slices/ShopCheckout/ShopCheckout.test.tsx:17]: Same question here.

File: src/slices/ShopCheckout/ShopCheckout.stories.tsx (hunk 1 above)
Comment (lialila) [reply src/slices/ShopCheckout/ShopCheckout.stories.tsx:87]: ok, i'll do it 20

File: src/slices/ShopCheckout/ShopCheckout.tsx [hunk 3]
```diff
@@ -160,76 +165,107 @@ export const ShopCheckout = ({ slice }: ShopCheckoutProps): JSX.Element => {
...
+                        )} CO₂`}
+                      />
+                      <LabelNumberPair
+                        label={formatMessage({
+                          id: 'sections.shopCheckout.summary.price',
+                        })}
+                        number={formatNumber(
+                          Math.floor(
+                            values.contributionValue / slice.pricePerKg
+                          ) * slice.pricePerKg,
```
Comment (rschiffer) [inline src/slices/ShopCheckout/ShopCheckout.tsx:227]:
```suggestion
                          summaryPrice,
```

File: src/slices/ShopCheckout/ShopCheckout.tsx [hunk 4]
```diff
@@ -160,76 +165,107 @@ export const ShopCheckout = ({ slice }: ShopCheckoutProps): JSX.Element => {
...
+                          slice.taxInPercent > 0 &&
+                          summaryPrice > 0
+                            ? formatMessage(
+                                {
+                                  id: 'sections.shopCheckout.summary.price.taxIncluded',
+                                },
+                                {
+                                  number: formatNumber(
+                                    summaryPrice +
+                                      (summaryPrice * slice.taxInPercent) / 100,
```
Comment (rschiffer) [inline src/slices/ShopCheckout/ShopCheckout.tsx:246]:
```suggestion
                                      summaryPrice * (slice.taxInPercent / 100),
```
I think it makes more sense like this regarding readibility

File: src/slices/ShopCheckout/ShopCheckout.tsx [hunk 5]
```diff
@@ -143,7 +146,9 @@ export const ShopCheckout = ({ slice }: ShopCheckoutProps): JSX.Element => {
                 unit: 'kilogram',
               })}  CO₂`}
             />
-
+            <Text size="smLowNormal">
+              <FormattedMessage id="sections.shopCheckout.summary.price.taxNotIncluded" />
+            </Text>
```
Comment (lukasbals) [inline src/slices/ShopCheckout/ShopCheckout.tsx:151]: This should be in the caption of the `LabelNumberPair`

**Review by rschiffer — APPROVED** [review]

**Pull request merged**